src/main/DAL/MetaQuestion/MetaQuestionRepository.js

```diff
@@ -21,7 +26,38 @@ class MetaQuestionRepository {
         this.#MetaQuestion.belongsToMany(this.#Keyword, {through: 'MetaQuestionKeyword', as: 'keywords'});
         this.#Keyword.belongsToMany(this.#MetaQuestion, {through: 'MetaQuestionKeyword', as: 'metaQuestions'});
 
-        sequelize.sync({alter: true}); // cleans the DB
+        // MQ - Appendix
+        this.#Appendix.hasMany(this.#MetaQuestion, {as: 'metaQuestions', foreignKey: 'appendixTag'});
+        this.#MetaQuestion.belongsTo(this.#Appendix, {as: 'appendix', foreignKey: { name: 'appendixTag', allowNull: true }});
+
+        // Appendix - Keyword
+        this.#Appendix.belongsToMany(this.#Keyword, {through: 'AppendixKeyword', as: 'keywords'});
```

Comment:
cant we create appendix and metaquestion without kewords?

cant we create a keyword that will assign only to metaquestion without appendix?

Reply:
1. yes we can, pass the empty array [] and it'll be created without keywords
2. yes we can, yes we can. this relation is defined this way solely for the option of recycling keywords